### ISSUE TITLE
Question about versioning of sly going forward

### ISSUE THREAD

**Issue by jpsnyder** [dabeaz/sly#104]
Hi, it is sad to hear that this project isn't being maintained anymore, but I understand. 

I have a question about the safety of the versions currently uploaded to PyPi. I understand you won't be pushing up anything new to it anymore, but will the versions currently up stay there for good? I ask because I have an important project which pulls from this during deployment. Pulling directly from github isn't feasible. So trying to determine if I should look into vendoring a copy of sly into my project.

Also do you have any thoughts on submitting sly to jazzband? https://jazzband.co/

**Comment by dabeaz** [comment]
It's not my intent to change anything about existing releases on PyPi.    For the long term future, I might suggest vendoring a copy of SLY--it has no dependencies other than Python standard library. 

I will probably continue to make periodic changes to SLY, but on a highly unpredictable and only occasional schedule.

**Comment by bjourne** [comment]
I'm also saddened by this news. I've used Sly for many projects and it is one of the most user friendly lexer/parser generator tools in existence.

**Comment by jpsnyder** [comment]
Yeah, was hoping the project could be taken under the wing of the jazzband group. That way we could still see new feature support and improvements.

**Comment by dabeaz** [comment]
There is nothing that prevents continued use of SLY or the addition of new features.  However, if you choose to use it, you need to vendor it into your application.    Not every library needs to be part of PyPi.   Sometimes the "free" in "free software" means "free as in puppy."   That is, you're free to take this puppy home.    Care and feeding at your house is your responsibility however.

**Comment by jpsnyder** [comment]
Okay, I'll ask again though. Any thoughts on releasing the project to https://jazzband.co/ ?
Or do you want to maintain ownership?

**Comment by dabeaz** [comment]
SLY is still something I may work on for myself.   I'm not interested in donating it.
